core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -1087,6 +1088,19 @@ public function makeRedirectPageMandatory(DataContainer $dc)
 		}
 	}
 
+	/**
+	 * Set useSSL to HTTP if the current request does not use HTTPS
+	 */
+	public function setUseSslDefault(DataContainer $dc)
```

Comment:
this should be a listener and not a DCA method

Reply:
Done in 30036212ba38b55b4d3a30d1d669b59e462287b6